src/dir.targets

```diff
@@ -10,6 +10,16 @@
   <Target Name="GetAssemblyVersion"
           Returns="$(AssemblyVersion)"/>
 
+  <!-- This Build target is only run if build tools have not been restored.  Once build tools have been restored, these gets overridden 
+       by the build tools import for the Build target. -->
+  <Target Name="Build" Condition="!Exists('$(ToolsDir)Build.Common.targets')" >
+    <CallTarget Targets="_RestoreBuildTools" />
```

Comment:
Weren't you going to add a message here about falling back to this target? Or is that message coming out of _RestoreBuildTools itself?